src/main/java/frc/utils/digitalinput/supplied/SuppliedDigitalInput.java

```diff
@@ -0,0 +1,40 @@
+package frc.utils.digitalinput.supplied;
+
+import edu.wpi.first.math.filter.Debouncer;
+import frc.utils.digitalinput.DigitalInputInputsAutoLogged;
+import frc.utils.digitalinput.IDigitalInput;
+
+import java.util.function.Supplier;
+
+public class SuppliedDigitalInput implements IDigitalInput {
+
+	private final Debouncer debouncer;
+	private boolean inverted = false;
+
+	private final Supplier<Boolean> isTrueSupplier;
+
+	public SuppliedDigitalInput(
+		Supplier<Boolean> isTrueConsumer,
```

Comment:
```suggestion
		Supplier<Boolean>booleanSupplier,
```

Reply:
maybe same as in the chooser, a name like inputValueSupplier or even without the word supplier